File: MiniDOMWebCtrls/DMD/Anagrafica/TipiRapportoModuleHandler.cs

namespace minidom.Forms
{
    public class TipiRapportoModuleHandler : CBaseModuleHandler
    {
        public TipiRapportoModuleHandler() : base(ModuleSupportFlags.SAnnotations | ModuleSupportFlags.SCreate | ModuleSupportFlags.SDelete | ModuleSupportFlags.SDuplicate | ModuleSupportFlags.SEdit)
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            return new Anagrafica.CTipoRapportoCursor();
        }

        public override object GetInternalItemById(int id)
        {
            return Anagrafica.TipiRapporto.GetItemById(id);
        }
    }
}

File: MiniDOMWebCtrls/DMD/Anagrafica/TipiRapportoModuleHandler.vb
Imports minidom.Anagrafica
Imports minidom.Databases

Namespace Forms

    Public Class TipiRapportoModuleHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            MyBase.New(ModuleSupportFlags.SAnnotations Or ModuleSupportFlags.SCreate Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SDuplicate Or ModuleSupportFlags.SEdit)

        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Return New CTipoRapportoCursor
        End Function

        Public Overrides Function GetInternalItemById(id As Integer) As Object
            Return Anagrafica.TipiRapporto.GetItemById(id)
        End Function

    End Class



End Namespace